### PR TITLE
Editor: update visual display of "code" block.

### PR DESCRIPTION
**Pull request by mtias**
Also update the font used.

Before:
image

After:
image

cc @aduth

### CODE REVIEW

File: client/components/tinymce/iframe.scss [hunk 1]
```diff
@@ -30,6 +30,12 @@ a:active {
 	outline: 0;
 }
 
+code  {
```
Comment (aduth) [inline client/components/tinymce/iframe.scss]: Extra space 😄

**Comment by aduth** [comment]
I think this looks great for inline code snippets. I worry the affect its had on `<pre><code>` multi-line code blocks:

`master`|`update/editor-code-block-css`
---|---
master|update

To be fair, we should really seek to improve the look of these blocks in any case.

**Comment by aduth** [comment]
Should be simple enough to apply styling to `pre` using the same background and adjusted padding, removing the padding from any descendent `code` blocks. Example:

image

**Comment by mtias** [comment]
That sounds good to me.

**Comment by aduth** [comment]
Looks good now across these three use-cases:
- Inline code text
- Preformatted code text
- Preformatted non-code text

Last thought would be... if we're worried about preformatted background being too dark, should we at least be consistent and apply the same lighter background to the inline code element? I feel it'd still look nice with the lighter gray. Your call though.

Otherwise, good to merge 👍

**Comment by mtias** [comment]
I feel it may be too dark for things like preformatted poetry, which seems to be a thing. But for code I think it's fine.

**Pull request merged**